spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/cache/CacheMeterBinderProvidersConfiguration.java

```diff
@@ -44,6 +47,17 @@
 @ConditionalOnClass(MeterBinder.class)
 class CacheMeterBinderProvidersConfiguration {
 
+	@Configuration(proxyBeanMethods = false)
+	@ConditionalOnClass({ SpringCache2kCache.class, Cache2kBuilder.class, org.cache2k.extra.micrometer.Cache2kCacheMetrics.class })
```

Comment:
`Cache2kCacheMetrics` doesn't need to be fully qualified unless I am missing something.